pom.xml

```diff
@@ -730,6 +730,15 @@
 				</plugins>
 			</build>
 		</profile>
+		<profile>
```

Comment:
Is this change reverent to this pr(bump client spring boot version)?

Reply:
reverted, I use this profile to test on my laptop